rlclientlib/ranking_event.h

```diff
@@ -170,4 +172,47 @@ namespace reinforcement_learning {
     bool _action_taken = false;
     unsigned int _outcome_type = 0;
   };
+
+  enum payload_type {
+    CB = 1,
+    CCB = 2,
+    SLATES = 3,
+    OUTCOME_SINGLE = 4
```

Comment:
how is an activation event encoded?
will this handle the logging interface you added?